packages/react-core/src/components/Tabs/__tests__/OverflowTab.test.tsx

```diff
@@ -0,0 +1,401 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import userEvent from '@testing-library/user-event';
+import { OverflowTab } from '../OverflowTab';
+import { TabsContext } from '../TabsContext';
+
+jest.mock('../../../helpers', () => ({
+  Popper: ({ trigger, popper, isVisible, popperMatchesTriggerWidth, appendTo }) => (
+    <div data-testid="popper-mock">
+      <div data-testid="trigger">{trigger}</div>
+      <div data-testid="popper">{isVisible && popper}</div>
+      <p>Popper matches trigger width: {`${popperMatchesTriggerWidth}`}</p>
+      <p>Append to class name: {appendTo && `${appendTo.className}`}</p>
+    </div>
+  )
+}));
+
+jest.mock('../../Menu', () => {
+  const { forwardRef } = jest.requireActual('react');
+  return {
+    Menu: forwardRef((props: { children; onSelect }, ref) => (
+      <div data-testid="menu-mock" ref={ref}>
+        {props.children}
+        <button onClick={() => props.onSelect('mouseEvent', 1, 'fakeRef')}>Select</button>
+      </div>
+    )),
+    MenuContent: ({ children }) => <div data-testid="menu-content-mock">{children}</div>,
+    MenuList: ({ children }) => <ul data-testid="menu-list-mock">{children}</ul>,
+    MenuItem: ({ children, itemId, isSelected }) => (
+      <>
+        <li id={`${itemId}-menu-item-mock`}>{children}</li>
+        <p>Selected: {`${isSelected}`}</p>
```

Comment:
Nit:
```suggestion
        <li id={`${itemId}-menu-item-mock`}>
          {children}
          <p>Selected: {`${isSelected}`}</p>
        </li>
```
Or something along these lines, just to have valid HTML in the test. Overall may not be a huge thing since it's a test so not blocking over it, but let me know what you think in regards to catching this sort of stuff in future tests.

Reply:
I think that's a great suggestion 🙂